File: tests/scripts/clusterProfiler/clusterProfiler.R
library(clusterProfiler)


genelist <- c(8318, 991, 9133, 890, 983, 4085, 7272, 1111, 891, 4174, 9232)

res <- enrichKEGG(gene = genelist, organism = "hsa", pvalueCutoff = 0.05)
df <- as.data.frame(res)

write.csv(df, "result.csv")

File: tests/scripts/clusterProfiler/clusterProfiler.py
from rwrap import clusterProfiler, base


genelist = [8318, 991, 9133, 890, 983, 4085, 7272, 1111, 891, 4174, 9232]

res = clusterProfiler.enrichKEGG(gene=genelist, organism="hsa", pvalueCutoff=0.05)
df = base.as_data_frame(res)

df.to_csv("result.csv")
